api/clients/codecs/fft_test.go

```diff
@@ -0,0 +1,39 @@
+package codecs
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/Layr-Labs/eigenda/common/testutils/random"
+	"github.com/stretchr/testify/require"
+)
+
+// TestFFT checks that data can be IFFTed and FFTed repeatedly, always getting back the original data
+func TestFFT(t *testing.T) {
```

Comment:
```suggestion
// TestFFT checks that data can be IFFTed and FFTed repeatedly, always getting back the original data
// TODO: we should probably be using fuzzing instead of this kind of ad-hoc random search testing
func TestFFT(t *testing.T) {
```